Apps/Encounter/src/appsettings.hgpoc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "HealthGateway": "Trace"
+        }
+    },
+    "OpenIdConnect": {
+        "Authority": "https://dev.oidc.gov.bc.ca/auth/realms/ff09qn3f"
+    },
+    "PatientService": {
+        "ClientRegistry": {
+            "ServiceUrl": "https://hiat2.hcim.ehealth.gov.bc.ca/HCIM.HIALServices.Portal/QUPA_AR101102.asmx",
```

Comment:
just in case the Patient Service URL is not this one :-)

Reply:
The POC environment is identical to DEV except for the URL